.github/workflows/build-test.yml

```diff
@@ -43,6 +43,8 @@ env:
   IPYWIDGET_SCREENSHOT_PATH: '*-screenshot.png'
   DISABLE_INSIDERS_EXTENSION: 1 # Disable prompts to install Insiders in tests (else it blocks activation of extension).
   VSC_JUPYTER_INSTRUMENT_CODE_FOR_COVERAGE: true
+  VSIX_NAME_PYTHON: 'ms-python-insiders.vsix'
+  VSC_JUPTYER_PYTHON_EXTENSION_VERSION: 'stable'
```

Comment:
Does this mean, if this is stable, then we use Stable VS Code & stable Python extension
& if its insider we use vscode insiders & python insiders?

Reply:
No it's just a flag for now. It makes it easy to switch this later.